migrations/2021-08-23-161500_aggregated_ft_nft_tables/up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE aggregated__fungible_tokens
+(
+    id                                bigserial                NOT NULL,
```

Comment:
I feel that `id` is useless. We should have some way to define an order of events, e.g. `processing_index_in_chunk` (`ORDER BY processed_in_block_timestamp, processing_index_in_chunk`  we don't care if tokens have the same `processing_index_in_chunk` since the receipts in the same block do not overlap and even more so, the receipts to the same FT/NFT token contract are bounded to a single shard where the contract resides)

Reply:
I am OK with adding `processing_index_in_chunk` for sorting purposes, but I'm still not sure is it a good idea to get rid of the artificial `id` column. 

Some thoughts on why having `id` as PK is good:
- I still think about duplicated tx hashes and maybe it's better to gradually migrate to artificial `id` as PK everywhere. In my opinion, our first task is to show data, only after it we can validate it somehow. So, if `timestamp` + `index_in_chunk` is not unique for some reason, it's better to add the value anyway, and then decide what should we do with it.
- If the PK is large, you have fewer keys per page in the index and the index will take more cache memory to store it. We have 8 bytes (one id bigserial) VS 20 bytes (16 bytes in timestamp numeric(20,0) and 4 bytes in index_in_chunk integer). Performance is also degraded because we have to scan more memory.
- Comparing numerics could be more expensive than comparing integers. I'm not sure about it, it is not mentioned in the documentation in a straight way, but 5-minute googling gives this thought.